internal/blocktx/processor.go

```diff
@@ -675,7 +683,7 @@ func (p *Processor) markTransactionsAsMined(ctx context.Context, blockId uint64,
 	// update all remaining transactions
 	updateResp, err := p.store.UpsertBlockTransactions(ctx, blockId, txs)
 	if err != nil {
-		return fmt.Errorf("failed to insert block transactions at block height %d: %v", blockHeight, err)
```

Comment:
Is it ok that we lose block height information in the error?

Reply:
It would indeed be nice to keep it. 
In other places like here: https://github.com/bitcoin-sv/arc/blob/0ab0a1f378d207467e7c7803830200a067106757/internal/beef/beef.go#L177
it was solved like this

```go
errors.Join(ErrFailedToInsertBlockTransactions, fmt.Errorf("block height: %d", blockHeight), err)
```

Reply:
Added block height